### ISSUE TITLE
In API with headless=false, can only see people list

### ISSUE THREAD

**Issue by lascimmia** [open-wa/wa-automate-nodejs#2135]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### Current Behavior

As object; the API works good, but since yesterday I'm unable to use WhatsApp Web while wa-automate API are running.
I can't see neither write in the message area on the right, just the people list on the left.
See the screenshot for details.

### Expected Behavior

As was working 2 days ago, allowing me to use WAWeb normally during API use

### Mode

EASY API/CLI

### DEBUG INFO

```markdown
{
  "WA_VERSION": "2.2136.9",
  "WA_AUTOMATE_VERSION": "4.21.1",
  "BROWSER_VERSION": "Chrome/93.0.4577.0",
  "START_TS": 1631772075369
}
```

### Environment

```markdown
- OS: Windows 10
- Node: 14.17.6
- npm: 6.14.15
```

### Screenshots

![SSOWA](https://user-images.githubusercontent.com/71778736/133560196-c42da998-3537-4d8e-aa0a-bd3b78f3cb6b.png)

**Comment by lascimmia** [comment]
The normal behaviour is like this:

![immagine](https://user-images.githubusercontent.com/71778736/133560690-5ac61666-c310-4619-bac3-9726f8af2c1f.png)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.22.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.22.0...4.22.1) (2021-09-19)

- build(deps): bump cfonts from 2.9.3 to 2.10.0 [`#2147`](https://github.com/open-wa/wa-automate-nodejs/pull/2147)
- Release 4.22.1 [`c3072f2`](https://github.com/open-wa/wa-automate-nodejs/commit/c3072f297f53a2fe61e565491fac8e6c5c2eae14)
- 🐛 fix broken headful mode #2135 [`2c39d06`](https://github.com/open-wa/wa-automate-nodejs/commit/2c39d06d293dc24aa59997ae7b336c031a5f4e82)
- updated types-only package [`c16a506`](https://github.com/open-wa/wa-automate-nodejs/commit/c16a50661d06006e9dfa8df54a2f0a4241474e59)

**Comment by smashah** [comment]
@lascimmia this is a very hard problem to solve but the latest update has some features to help prevent the issue.

The one case in which you will still experience this error is when scanning the qr code the first time. 

If you absolutely need the session to work properly in headful mode even on the qr scan then set [`ensureHeadfulIntegrity: true`](https://docs.openwa.dev/interfaces/api_model_config.ConfigObject.html#ensureHeadfulIntegrity) or `--ensure-headful-integrity` which will refresh your session upon creation. It will increase your load times.

thanks
